src/pages/quiz-page/quiz-page.component.tsx

```diff
@@ -0,0 +1,91 @@
+import { useForm } from 'react-hook-form';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ROUTES } from '../../shared/constants/routes.constants';
+import { Logo } from '../../shared/logo/logo.component';
```

Comment:
nice to have: Use aliases if it's available

Reply:
Okey,i'll try